docs/content/themes/float/index.md

```diff
@@ -46,7 +46,8 @@ Float 是一款為 [Zola](https://www.getzola.org/) 設計的佈景主題。
 在您的 Zola 專案資料夾內：
 
 把 Float 以 Git 子模組的方式加入專案內：
-```shell
+
+```sh
```

Comment:
This is from a theme so it will get overriden the next time a theme is updated and I should probably not include non-English themes...